src/components/Inputs/CustomInputs.jsx

```diff
@@ -0,0 +1,34 @@
+import './CustomInputs.scss'
+
+
+
+function CustomInput ({border, width, height,type, id, label, placeholder, className,classNameLabel}) {

```

Comment:
```suggestion
function CustomInput ({border, width, height,type = 'text', id, label, placeholder, className,classNameLabel}) {
```

Você pode adicionar valores default para as props assim, ao invés de verificar se existe um valor lá embaixo